{
  "UID": "NC:werapi.PFN_WER_RUNTIME_EXCEPTION_EVENT",
  "title": "PFN_WER_RUNTIME_EXCEPTION_EVENT (werapi.h)",
  "description": "Windows Error Reporting (WER) calls this function to determine whether the exception handler is claiming the crash.",
  "helpviewer_keywords": [
    "OutOfProcessExceptionEventCallback",
    "OutOfProcessExceptionEventCallback callback function [Windows Error Reporting]",
    "PFN_WER_RUNTIME_EXCEPTION_EVENT",
    "PFN_WER_RUNTIME_EXCEPTION_EVENT callback",
    "wer.outofprocessexceptioneventcallback",
    "werapi/OutOfProcessExceptionEventCallback"
  ],
  "old-location": "wer\\outofprocessexceptioneventcallback.htm",
  "tech.root": "wer",
  "ms.assetid": "22033278-2be3-4621-b618-3ccd21fb4cdd",
  "ms.date": "07/21/2023",
  "ms.keywords": "OutOfProcessExceptionEventCallback, OutOfProcessExceptionEventCallback callback function [Windows Error Reporting], PFN_WER_RUNTIME_EXCEPTION_EVENT, PFN_WER_RUNTIME_EXCEPTION_EVENT callback, wer.outofprocessexceptioneventcallback, werapi/OutOfProcessExceptionEventCallback",
  "req.header": "werapi.h",
  "req.include-header": null,
  "req.target-type": "Windows",
  "req.target-min-winverclnt": "Windows 7 [desktop apps only]",
  "req.target-min-winversvr": "Windows Server 2008 R2 [desktop apps only]",
  "req.kmdf-ver": null,
  "req.umdf-ver": null,
  "req.ddi-compliance": null,
  "req.unicode-ansi": null,
  "req.idl": null,
  "req.max-support": null,
  "req.namespace": null,
  "req.assembly": null,
  "req.type-library": null,
  "req.lib": null,
  "req.dll": null,
  "req.irql": null,
  "targetos": "Windows",
  "req.typenames": null,
  "req.redist": null,
  "ms.custom": "19H1",
  "f1_keywords": [
    "PFN_WER_RUNTIME_EXCEPTION_EVENT",
    "werapi/PFN_WER_RUNTIME_EXCEPTION_EVENT"
  ],
  "dev_langs": [
    "c++"
  ],
  "topic_type": [
    "APIRef",
    "kbSyntax"
  ],
  "api_type": [
    "UserDefined"
  ],
  "api_location": [
    "Werapi.h"
  ],
  "api_name": [
    "OutOfProcessExceptionEventCallback"
  ]
}